### PR TITLE
add JuliaCon22 blogpost

### PR DESCRIPTION
**Pull request by tmigot**

### CODE REVIEW

**Review by geoffroyleconte — APPROVED** [review]

File: news-and-blogposts/2022/2022-07-30-juliacon22.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@def title = "Conference: JuliaSmoothOptimizers at JuliaCon 2022"
+@def rss_description = "This year JuliaCon 2022/ JuMP-dev 2022 featured three talks on JuliaSmoothOptimizers."
+
+# Conference: JuliaCon 2022
+
+This year [JuliaCon 2022](https://juliacon.org/2022)/ JuMP-dev 2022 featured three talks on JuliaSmoothOptimizers during the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.
```
Comment (abelsiqueira) [inline news-and-blogposts/2022/2022-07-30-juliacon22.md:6]:
```suggestion
This year's [JuliaCon 2022/JuMP-dev 2022](https://juliacon.org/2022) featured three talks on JuliaSmoothOptimizers during the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.
```

**Review by abelsiqueira — APPROVED** [review]
Thanks, Tangi! I made a small suggestion, it's approved otherwise.

File: news-and-blogposts/2022/2022-07-30-juliacon22.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@def title = "Conference: JuliaSmoothOptimizers at JuliaCon 2022"
+@def rss_description = "This year JuliaCon 2022/ JuMP-dev 2022 featured three talks on JuliaSmoothOptimizers."
```
Comment (dpo) [inline news-and-blogposts/2022/2022-07-30-juliacon22.md:2]:
```suggestion
@def rss_description = "This year's JuliaCon 2022/ JuMP-dev 2022 featured three talks on JuliaSmoothOptimizers."
```

File: news-and-blogposts/2022/2022-07-30-juliacon22.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+# Conference: JuliaCon 2022
+
+This year's [JuliaCon 2022/JuMP-dev 2022](https://juliacon.org/2022) featured three talks on JuliaSmoothOptimizers during the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.
```
Comment (dpo) [inline news-and-blogposts/2022/2022-07-30-juliacon22.md:6]:
```suggestion
This year's [JuliaCon 2022/JuMP-dev 2022](https://juliacon.org/2022) featured three talks on JuliaSmoothOptimizers in the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.
```

File: news-and-blogposts/2022/2022-07-30-juliacon22.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Dominique Orban gave a complete overview in 20 minutes (!!) of the JuliaSmoothOptimizers organization. The talk mentions all the key packages to model optimization problem ([NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl) & co), solve the problem ([JSOSolvers.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl) & co) and do simple yet amazing benchmarks with [SolverBenchmark.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl).
```
Comment (dpo) [inline news-and-blogposts/2022/2022-07-30-juliacon22.md:8]: 25 minutes?

Comment (dpo) [inline news-and-blogposts/2022/2022-07-30-juliacon22.md:8]:
```suggestion
Dominique Orban gave a complete overview in 20 minutes (!!) of the JuliaSmoothOptimizers organization. The talk mentions all the key packages to model optimization problems ([NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl) & Co), solve models ([JSOSolvers.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl) & Co), and do simple yet amazing benchmarks with [SolverBenchmark.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl).
```

File: news-and-blogposts/2022/2022-07-30-juliacon22.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Geoffroy Leconte presented [RipQP.jl](https://github.com/JuliaSmoothOptimizers/RipQP.jl) a multi-precision algorithm for convex quadratic optimization. The multi-precision feature illustrates the strength of pure Julia implementations, and the performance is there.
```
Comment (dpo) [inline news-and-blogposts/2022/2022-07-30-juliacon22.md:10]:
```suggestion
Geoffroy Leconte presented [RipQP.jl](https://github.com/JuliaSmoothOptimizers/RipQP.jl), a multi-precision algorithm for convex quadratic optimization. The multi-precision feature illustrates the strength of pure Julia implementations, and the ensuing performance.
```

File: news-and-blogposts/2022/2022-07-30-juliacon22.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+Tangi Migot showed an application of the JuliaSmoothOptimizers framework for PDE-constrained optimization modeled by [PDENLPModels.jl](https://github.com/JuliaSmoothOptimizers/PDENLPModels.jl).
```
Comment (dpo) [inline news-and-blogposts/2022/2022-07-30-juliacon22.md:12]:
```suggestion
Tangi Migot showed an application of the JuliaSmoothOptimizers framework to PDE-constrained optimization problems modeled with [PDENLPModels.jl](https://github.com/JuliaSmoothOptimizers/PDENLPModels.jl).
```

**Review by dpo — CHANGES_REQUESTED** [review]

**Comment by dpo** [comment]
Thank you! All good to go after the minor changes above!

**Pull request merged**